ddtrace/internal/telemetry/writer.py

```diff
@@ -429,6 +429,14 @@ def _app_started_event(self, register_app_shutdown=True):
                 "message": self._error[1],
             },
         }  # type: Dict[str, Union[Dict[str, Any], List[Any]]]
+        # Add time to value telemetry metrics for single step instrumentation
+        if config._telemetry_install_id or config._telemetry_install_type or config._telemetry_install_time:
+            payload["install_signature"] = {
+                "install_id": config._telemetry_install_id,
```

Comment:
`None` values are ok here? since as long as one isn't `None` they will all be included.

Reply:
I ran a quick check and yes, they are fine. Python will serialize them to `null` in JSON and Go on our backend will interpret them as empty strings, so whether they are set to None or not included in the map at all, the backend will do the same thing.